station/tests/test_quality_ratings.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+
+import numpy as np
+
+import quality_ratings
+
+
+class TestQualityRatings(unittest.TestCase):
+    def test_list_names(self):
+        names = quality_ratings.get_rate_names()
+        self.assertTrue("analog" in names)
+        self.assertTrue("digital" in names)
+
+    def test_get_rate_by_name(self):
+        self.assertIsNotNone(quality_ratings.get_rate_by_name("analog"))
+        self.assertIsNotNone(quality_ratings.get_rate_by_name("digital"))
+
+    def test_analog_rating_on_gaussian_noise_small_sigma(self):
+        img = np.random.normal(scale=1, size=(1000, 1000))
```

Comment:
I don't understand how this test is working. Some sort of an explanation is necessary. The code generates 1M values with Gauss distribution. Since loc is not specified, the peak is at 0.0 and the samples can take both positive and negative values. Why does this imply that the rating is almost exactly 1.0 (perfect image)?

Reply:
The estimator try to estimate the sigma of Gauss distribution. Next the estimated sigma is used to calculate finish rate. Based on empirical estimations I set the default parameters. Sigma < 10 means excellent image. Sigma > 20 means completely noisy image.  Range between 10 and 20 is cast on normal distribution curve. 

For details check [this section](https://github.com/gut-space/analog-noise-estimator-cli#gauss-rating).

Reply:
ok